### ISSUE TITLE
Do not log an error when CMS is returning an HTTP error

### ISSUE THREAD

**Issue by benoit74** [openzim/zimfarm#825]
When performing a query to the CMS, it happens that an error occurs in the normal processing (might happen) AND in the error handling (not normal), leading to a fatal issue.

```
2023-09-22T05:32:19.757990065Z stderr F DEBUG:urllib3.connectionpool:https://api.cms.openzim.org:443 "POST /v1/books/add HTTP/1.1" 413 176
2023-09-22T05:32:19.758388859Z stderr F ERROR:common.external:Unable to parse CMS response: Expecting value: line 1 column 1 (char 0)
2023-09-22T05:32:19.758842003Z stderr F ERROR:common.external:Expecting value: line 1 column 1 (char 0)
2023-09-22T05:32:19.758847505Z stderr F Traceback (most recent call last):
2023-09-22T05:32:19.758850657Z stderr F   File "/usr/local/lib/python3.8/site-packages/requests/models.py", line 971, in json
2023-09-22T05:32:19.758853333Z stderr F     return complexjson.loads(self.text, **kwargs)
2023-09-22T05:32:19.758855235Z stderr F   File "/usr/local/lib/python3.8/json/__init__.py", line 357, in loads
2023-09-22T05:32:19.758856804Z stderr F     return _default_decoder.decode(s)
2023-09-22T05:32:19.758866868Z stderr F   File "/usr/local/lib/python3.8/json/decoder.py", line 337, in decode
2023-09-22T05:32:19.758868632Z stderr F     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
2023-09-22T05:32:19.758870374Z stderr F   File "/usr/local/lib/python3.8/json/decoder.py", line 355, in raw_decode
2023-09-22T05:32:19.758872071Z stderr F     raise JSONDecodeError("Expecting value", s, err.value) from None
2023-09-22T05:32:19.758873621Z stderr F json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2023-09-22T05:32:19.758875112Z stderr F 
2023-09-22T05:32:19.75887671Z stderr F During handling of the above exception, another exception occurred:
2023-09-22T05:32:19.758878238Z stderr F 
2023-09-22T05:32:19.758879849Z stderr F Traceback (most recent call last):
2023-09-22T05:32:19.758881454Z stderr F   File "/app/./common/external.py", line 198, in advertise_book_to_cms
2023-09-22T05:32:19.75888322Z stderr F     data = resp.json()
2023-09-22T05:32:19.758884877Z stderr F   File "/usr/local/lib/python3.8/site-packages/requests/models.py", line 975, in json
2023-09-22T05:32:19.758886444Z stderr F     raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
2023-09-22T05:32:19.758888039Z stderr F requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by benoit74** [comment]
Indeed, this is "normal", it is just the full log of the exception which occurred, the `requests` package is obviously not supporting our use case very nicely, but no fatal exception are raised in fact.

**Comment by rgaudin** [comment]
OK, so we're not re-raising from there?

**Comment by benoit74** [comment]
I think we are not re-raising, yes, code is quite ok

**Comment by benoit74** [comment]
Fixed by #827
